docs/03-Filtering Patches.md

```diff
@@ -1,18 +1,51 @@
 ## Filtering Patches
 
-You can ignore local changes by setting `Palindrom.filterLocalChange` function. This function should accept a JSON Patch operation and either return it or return a falsy value. Obviously, returning a falsy value will result an ignored operation.
+### 1. By defining non-enumerable properties _(recommended)_:
+Assuming your Palindrom instance is instantiated and your `palindrom.obj` is ready, you can create a non-enumerable object child in `palindrom.obj` and use it as a local non-synced store for your needs.
+
+#### Example:
+
+```js
+Object.defineProperty(palindrom.obj, "myLocalNonSyncedObject", {
+    enumerable: false,
+    value: {},
+    writable: true
+});
+```
+Now `palindrom.obj.myLocalNonSyncedObject` is an object this is not synced with the server. This is ideal for keeping client-side related information (eg: div height, or something like `isModalWarningDisplayed = true`).
+
+Example:
+
+```js
+palindrom.obj.myLocalNonSyncedObject.divHeight = 50;
+```
+... is a local change.
+
+**Note:  The added property must be an object.***
+
+_*Adding a non-enumerable property adds a property to the object in a non-synced manner, however, if you modify this property Palindrom will nonetheless try to sync it. To overcome that, the added property must be an object that doesn't change itself, only its children do, and that doesn't trigger Palindrom._
+
+### 2. By extending `Palindrom#filterLocalChange`
+
+You can ignore local changes by exteding `Palindrom.filterLocalChange` function. This function should accept a JSON Patch operation and either return it or return a falsy value. Obviously, returning a falsy value will result an ignored operation.
```

Comment:
> You can ignore local changes

You can discard a generated patch

> by exteding

by extending

> This function should accept a JSON Patch operation and either return it or return a falsy value. Obviously, returning a falsy value will result an ignored operation.

This function accepts a JSON Patch operation and either returns it or returns a falsy value. Returning a falsy value results in ignoring this operation.